tests/unit/crawlers/_beautifulsoup/test_beautifulsoup_crawler.py

```diff
@@ -204,23 +204,29 @@ async def request_handler(context: BeautifulSoupCrawlingContext) -> None:
 
 
 @pytest.mark.parametrize(
-    ('queue_name', 'queue_alias'),
+    ('queue_name', 'queue_alias', 'by_id'),
```

Comment:
Just for consistency, could the order please be id, name, alias?

Also, as I mentioned above, keep the name of the params consistent.